### PR TITLE
New Hai Phong fashion - Adds an ao dai, resprites the non la

### PR DESCRIPTION
**Pull request by atteria**
Just in time for Tet on the 22nd.
Adds an ao dai, a traditional Vietnamese garment. The high side split might look weird, but it's always worn over pants.
The old non la sprite had pretty clear jaggies and had some weird texture. 

There are some related sprites not in this PR I still have in reserve - an ao yem(added now), a changshan, a customizable respirator, and a sandstorm veil. If there's interest in any of these I can add them, in this PR or another.

nhp showcase

### CODE REVIEW

**Comment by atteria** [comment]
!review

**Comment by alsoandanswer** [comment]
Some comments :
I know this isn't realistic, but you should group the ao dais together as a singular item, as rolled sleeve variants. (It's not *literal* rolled sleeves, just the name of the function.
This is mainly to avoid code and loadout bloat. (Imagine if every jumpsuit had a variant for rolled down and rolled sleeves!))
Speaking of which, should probably add a male variant like the konyang hanboks, to keep consistent with our clothing having either both gender or unisex clothing (unless explicitly being for one gender only!)

As for the rest of the sprites, I think it should be fine to add them. Clothing PRs aren't usually very code intensive to review.

**Comment by atteria** [comment]
It would be unintuitive and odd for an item to go from ankle-length to knee-length through the sleeve rolling mechanic. I think it would be unclear to players that there are two different dress cuts here with different sprites and lore context. 

It wouldn't get rid of the type list either if I add a masculine version. Which is a good idea, there should be.

**Comment by alsoandanswer** [comment]
That's reasonable. Carry on, then.

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -559,3 +559,16 @@
 	flagpatch_national["flagpatch, nralakk"] = /obj/item/clothing/accessory/flagpatch/nralakk
 	flagpatch_national["flagpatch, hegemony"] = /obj/item/clothing/accessory/flagpatch/hegemony
 	gear_tweaks += new /datum/gear_tweak/path(flagpatch_national)
+
+/datum/gear/accessory/aodai
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:563]: Are the ao dai and non la intentionally not culture restricted?

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (atteria) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:563]: The non la wasn't previously, and reasonably an Earth character could use either. Should I restrict to both there and NHP?

Comment (NonQueueingMatt) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:563]: Sure.

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**